plugins/woocommerce/client/legacy/js/frontend/checkout.js

```diff
@@ -541,12 +541,14 @@ jQuery( function( $ ) {
 						$( '.checkout-inline-error-message' ).remove();
 
 						try {
-							if ( 'success' === result.result && 
+							if ( 'success' === result.result &&
 								$form.triggerHandler( 'checkout_place_order_success', [ result, wc_checkout_form ] ) !== false ) {
-								if ( -1 === result.redirect.indexOf( 'https://' ) || -1 === result.redirect.indexOf( 'http://' ) ) {
-									window.location = result.redirect;
-								} else {
-									window.location = decodeURI( result.redirect );
+								if ( result.redirect ) {
```

Comment:
I would prefer to undo this change. All docs point out to `success` responses having a redirect, and I believe that it is always provided.

If, for any reason, the property is missing, I'd rather see an error in the console, rather than silent failure.

Reply:
That is a fair point, I reverted it here [5fb18f0](https://github.com/woocommerce/woocommerce/pull/53156/commits/5fb18f0a7e51a8610db6b7a65e9ec51525e1b09f) thanks!